Travel_fellowships.md

```diff
@@ -1,41 +1,183 @@
+For example, you may be hosting the event, giving a talk, chairing a session, running a Birds of a Feather discussion, offering a training event, facilitating a sprint (or similar session), leading a project at a code fest/hackathon, presenting a poster, participating in a panel discussion, reviewing abstracts for the event, contribute to the event as a co-organiser, or volunteer to help the organisation committee during the event.
+
+#### Open Source Resources
+
+The review panel may look at your general involvement in the open source development (such as your GitHub contributions to the Open Source project) or open science communities (for example, your previous work advocating for open practices and diversity).
+
+Any information regarding your past participation in the project that you are representing will be useful for evaluating your application as well.
+Therefore, please supply the link of the public repository or online resources that will help the panel in the review process.
+
+This will not be a limiting factor if you are new to bioinformatics and open science and express your enthusiasm and willingness to learn and practice open science principles learned from the event of your choice and OBF community.
+
+#### Promoting diversity and representation
+
+The review panel will look at how your attendance will contribute to increasing diversity at the event.
+We understand increasing diversity as increasing participation of members from groups traditionally underrepresented in the bioinformatics, computation or tech community or at the event.
+This includes but not limited to underrepresented demographic groups (members from many developing nations), ethnic origins, career stages, gender identity and expression, and people with disabilities.
+
+If you are not from a minority or underrepresented group but have a track record for advocating for diversity as an ally through your work or involvement in the event of your choice, you are welcome to apply for this fellowship as well.
+
+*We do not consider the diversity of operating systems, programming languages or text editors as a criterion for the OBF Travel fellowship.*
+
+## Announcement of the application outcome
+
+Applicants will be notified of the outcome no later than 3 weeks after the application deadline.
+
+Those who receive the fellowship award will be able to discuss any further details with the fellowship chair and/or the OBF board members.
+
+Applicants who are unsuccessful in a round will be provided with the feedback.
+They will be able to apply again with a revised and updated application for the next round of event fellowship.
+
+## Responsibilities of the fellowship awardees
+
+We expect that your participation in the event will promote open science practices in software development, training, documentation, publication and collaborative efforts in the biological research community, and will ideally promote diversity in their communities. 
```

Comment:
already said earlier, but maybe ok here as a summary

Reply:
I am adding it here so that I can share the link to this section directly with the awardees listing what is expected of them.